data/helpers.d/php

```diff
@@ -402,6 +402,12 @@ ynh_get_scalable_phpfpm () {
         php_max_children=$max_proc
     fi
 
+    # Get an potential forced value for php_max_children
+    local php_forced_max_children=$(ynh_app_setting_get --app=$app --key=php_forced_max_children)
+    if [ -n "$php_forced_max_children" ]; then
+        php_max_children=$php_forced_max_children
```

Comment:
Btw, shouldn't we check the value (could be done in config panel) to avoid negative values. We could do:

```suggestion
        php_max_children=$(at_least_one $php_forced_max_children)
```

Reply:
This function is used because we have division before, which could end up with unexpected values.
But $php_forced_max_children is from a value into the config panel, with a big warning saying that you shouldn't probably even change the value.
But If an user willingly turn that value to a negative one... Well, it will fail... And that will be a good lesson to have been so stupid...

Reply:
Well, it's a way of seeing things :smile:

Reply:
By the way, in parallel, I've added an action to reset the php-fpm config.
So even if an user fuck up his config, it can still reset to a default one (same for nginx and systemd. And also a last one... But I keep the suspense on that last one :wink:)

Reply:
hoo can't wait! :open_mouth: